frontend/awx/interfaces/InventorySource.ts

```diff
@@ -107,7 +107,8 @@ export interface InventorySourceForm {
   inventory?: number;
   source_path: { name: string | undefined | null };
   source_script?: string;
-  execution_environment?: SummaryFieldsExecutionEnvironment;
+  execution_environment?: string;
+  execution_environmentIdPath?: number;
```

Comment:
What's the different use cases for these 2 lines? Do we need them both?

Reply:
Yes it seems like it. We need the EE name string to be displayed in the search box but we need the EE ID number to be set when selecting an EE and submitting the data to the API. I'm not sure if this is the best way to do it, but it worked for the credential field so I just did the same here :)